extensions/amp-form/0.1/amp-form.js

```diff
@@ -443,7 +377,7 @@ function getUserValidityStateFor(element) {
 function checkUserValidity(element, propagate = false) {
```

Comment:
Isn't that the same thing? A form really isn't invalid unless you've told me it is.